### PR TITLE
CDS-1798 branch-Unstable tag

### PR DESCRIPTION
**Pull request by warcholprzemo**
handle `<branch>-Unstable` tag in builder list

### CODE REVIEW

**Comment by warcholprzemo** [comment]
List of builders. One builder has "old" `Unstable` and another has "new" `2018.2-Unstable`

branch-unstable

File: www/script/project/rtBuilders.js [hunk 1]
```diff
@@ -162,7 +179,7 @@ define(function (require) {
                         return b.toLowerCase() === branch_type.toLowerCase();
                     };
 
-                if (hideUnstable === true && ($.inArray(UNSTABLE_TAG, builderTags) > -1 || $.inArray(WIP_TAG, builderTags) > -1)) {
+                if (hideUnstable === true && (rtBuilders.hasUnstableTag(builderTags, branch_type) || $.inArray(WIP_TAG, builderTags) > -1)) {
```
Comment (campos-ddc) [inline www/script/project/rtBuilders.js:182]: You can remove all mention of the WIP_TAG, we don't use it anymore (feel free to do a separate PR for that if you prefer)

Comment (warcholprzemo) [reply www/script/project/rtBuilders.js:182]: I created new task with low priority: https://jira.hq.unity3d.com/browse/CDS-1870

File: www/script/project/rtBuilders.js [hunk 2]
```diff
@@ -150,6 +150,23 @@ define(function (require) {
 
             return selectedTags;
         },
+        hasUnstableTag: function(builderTags, branch_type){
+            for(var i=0; i<builderTags.length; i++){
+                var full_tag = builderTags[i];
+                if(full_tag.indexOf("-") > -1){
+                    var tag_branch = full_tag.split("-")[0];
+                    var tag_suffix = full_tag.split("-")[1];
```
Comment (Mowinski) [inline www/script/project/rtBuilders.js:158]: You can do it in future more simpler than two instruction
```
var [tag_branch, tag_suffix] = full_tag.split("-");
```

But its feature was introduced in javascript 1.7

File: www/script/project/rtBuilders.js [hunk 3]
```diff
@@ -150,6 +150,23 @@ define(function (require) {
 
             return selectedTags;
         },
+        hasUnstableTag: function(builderTags, branch_type){
+            for(var i=0; i<builderTags.length; i++){
```
Comment (Mowinski) [inline www/script/project/rtBuilders.js:154]: Keep in mind, that we should use array methods in javascript

**Review by Mowinski — APPROVED** [review]

**Pull request merged**